Classes/ViewHelpers/Form/SectionViewHelper.php

```diff
@@ -31,6 +31,8 @@ public function initializeArguments() {
 		$this->registerArgument('variables', 'array', 'Freestyle variables which become assigned to the resulting Component - ' .
 			'can then be read from that Component outside this Fluid template and in other templates using the Form object from this template', FALSE, array());
 		$this->registerArgument('extensionName', 'string', 'If provided, enables overriding the extension context for this and all child nodes. The extension name is otherwise automatically detected from rendering context.');
+		$this->registerArgument('inherit', 'boolean', 'If TRUE, the value for this particular field is inherited - if inheritance is enabled by the ConfigurationProvider', FALSE, TRUE);
```

Comment:
Default here should be FALSE as inheritance very rarely makes sense for Sections.